typescript-app/actions/tsAction/index.ts

```diff
@@ -0,0 +1,17 @@
+type Something = {
+  name: string;
+  age: number;
+};
+
+export async function main (params: any, thing: Something) {
```

Comment:
What's x for here, and does thing come through from rt? Or is it just example

Reply:
just demonstrating that types can be specified, and therefore TS ...